js/libs/keycloak-masthead/package.json

```diff
@@ -53,6 +53,7 @@
     "rollup-plugin-peer-deps-external": "^2.2.4",
     "vite": "^4.2.2",
     "vite-plugin-checker": "^0.5.6",
-    "vite-plugin-dts": "^2.3.0"
+    "vite-plugin-dts": "^2.3.0",
+    "vite-plugin-svgr": "^2.4.0"
```

Comment:
Do we really need to add a plugin to do SVGs? I'd rather we use one of the pre-defined components from PatternFly.